### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Blob File_BlobContainerAcquireLease_ScreenReader:Voiceover does not read about the locked status for which after click on the acquire lease

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1983]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver;#Mac;
**Environment Details:**
Desktop App: Windows Version: Windows10
MAC Version 10.14.6 Beta(18G29g)
**Steps to Reproduce:**
1.Launch Storage Explorer
2.Navigate to "Open Connect dialog" pane and Sign in to Azure.
3.Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4.Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5.Once completed, Navigate to "Upload" button and Upload a blob/file by using "Upload Files" control.
6.Right click on blob and select "Acquire Lease" control and hit enter.
7.Open Voiceover and check whether it reads anything about the locked status for which do the Acquire lease.
**Actual:**
NVDA does not read about the locked status for which after click on the acquire lease.
**Expected:**
NVDA should provide the information about locked symbol for which do the acquire lease
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**User Impact:**
Screen reader user unable to know that why the locked symbol is provided.
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference**
[1983_A11y_AzureToolsStorageExplorer_Blob File_BlobContainerAcquireLease_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3670218/1983_A11y_AzureToolsStorageExplorer_Blob.File_BlobContainerAcquireLease_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and MAC 10.15.3 and the issue is no longer reproduced.Hence close this issue.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
